### PR TITLE
Remove support for Hosted Graphite

### PR DESCRIPTION
**Pull request by simonplend**
Fixes issue #288.

**TODO**

- [x] Remove all code relating to Hosted Graphite
- [x] Fix all unit tests
- [x] Update README
- [x] Make FT Graphite hostname a constant to avoid hardcoding it everywhere
- [x] Document what `index.js` exports

### CODE REVIEW

File: lib/metrics.js [hunk 1]
```diff
@@ -43,24 +41,20 @@ Metrics.prototype.init = function (opts) {
 
 	this.opts = opts;
```
Comment (sjparkinson) [inline lib/metrics.js]: Doesn't exist?

Comment (simonplend) [reply lib/metrics.js]: Fixed.

**Comment by simonplend** [comment]
@sjparkinson Please re-review, especially the README!

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ A library for sending metrics to Graphite, that also provides drop in instrument
 
 ## Usage
 
-## Getting started
+### Getting started
```
Comment (sjparkinson) [inline README.md]: <3

**Pull request merged**